.cirrus.yml

```
@@ -0,0 +1,32 @@
+task:
+  matrix:
+    - name: FreeBSD
+      freebsd_instance:
+        image_family: freebsd-13-1
+    - name: Linux ARMv8
+      arm_container:
+        image: ubuntu:latest
+  env:
+    matrix:
+      - JULIA_VERSION: 1.3
+      - JULIA_VERSION: 1
+      - JULIA_VERSION: nightly
+  allow_failures: $JULIA_VERSION == 'nightly'
+  install_script: |
+    URL="https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh"
+    set -x
+    if command -v curl; then
+        echo "wget"
+        sh -c "$(curl ${URL})"
+    elif command -v wget; then
```

```suggestion
    if [ ! -z "$(which curl)" ]; then
        echo "wget"
        sh -c "$(curl ${URL})"
    elif [ ! -z "$(which wget)" ]; then
```
Literally no idea if this will help